crates/ruma-events/CHANGELOG.md

```diff
@@ -20,6 +20,7 @@ Improvements:
 * Add `room::message::MessageType::body` accessor method
 * Implement `Redact` for event structs (in addition to `Any` event enums)
 * Add `room::message::RoomMessageEventContent::{body, msgtype}` accessor methods
+* Move `room::message::MessageType::VerificationRequest`'s deserialization out of `unstable-pre-spec`
```

Comment:
I would like for this changelog entry to be rephrased. The enum variant itself was only recently un-feature-gated, looks like I missed the deserialization code but either way the change that should be documented (if any) is the enum variant being available without the flag.